pkg/overlay/service_test.go

```diff
@@ -0,0 +1,44 @@
+package overlay
+
+import (
+	"context"
+	"fmt"
+	"net"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"google.golang.org/grpc"
+	proto "storj.io/storj/protos/overlay" // naming proto to avoid confusion with this package
```

Comment:
how do you feel about separating storj imports into a third stanza, separate from the third party stanza? i think goimports has a flag for doing that

Reply:
I like it. fixed